src/main/java/mate/academy/mainspringproject/model/Book.java

```diff
@@ -33,4 +39,13 @@ public class Book {
     private String coverImage;
     @Column(name = "is_deleted")
     private Boolean isDeleted = false;
```

Comment:
```suggestion
    private boolean isDeleted = false;
```
I'm not sure NULL would make sense here